homeassistant/components/transmission/config_flow.py

```diff
@@ -37,7 +49,7 @@
 class TransmissionFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     """Handle Tansmission config flow."""
 
-    VERSION = 1
+    VERSION = CONF_ENTRY_VERSION
```

Comment:
Please increase only the minor version and leave the version as it is as this change is not a breaking change.
See https://developers.home-assistant.io/docs/config_entries_config_flow_handler#config-entry-migration

Reply:
Ahh ok, I missed that there was a minor attribute in the version for components when I created it, I'll try to work on it soon when I have some time! Thanks for the feedback btw! :pray:

Reply:
Updated accordingly!